server-sdk/src/main/java/com/amazonaws/ml/mms/servingsdk/http/Response.java

```diff
@@ -0,0 +1,9 @@
+package com.amazonaws.ml.mms.servingsdk.http;
+
+public interface Response {
+    void setStatus(int sc);
+    void setStatus(int sc, String phrase);
+    void setHeader(String k, String v);
```

Comment:
add: void addHeader() method

Reply:
Done. I didn't see a need for this for the first release, but I can add it.